README.md

```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
 community. Kops maintainers have agreed to set aside time specifically dedicated
 to working with newcomers, helping with PRs, and discussing new features.
 
-We recommend letting us know **beforehand** if you plan on attending so we can
-have time to prepare for the call.
+Please feel free to reach out **beforehand** if you plan on attending!
 
-| Maintainer   | Schedule      |  URL |
```

Comment:
Fine with removing it, but in general the maintainers are the ones expected to be on the call. Maintainers have different expectations from the community, and if you made the list you should be held to your commitment as a maintainer (in writing) IMHO.. so LGTM on this for now.. but would like to see a maintainers list pop up somewhere else if we could..